src/css/_no-requirements-toggle.scss

```
@@ -8,10 +10,68 @@ $gray: rgb(204, 204, 204);
   z-index: 900;
   padding: 0.2em 0.4em;
 
+  display: flex;
+  align-items: center;
+  gap: 1em;
+
   border: solid $gray 1px;
   border-radius: 2px;
 
   @media only screen and (min-width: 48em) {
     top: 11em;
   }
 }
+
+/* The switch - the box around the slider */
+.switch {
+  position: relative;
+  display: inline-block;
+  width: calc(3.75 * $toggle-size);
+  height: calc(2.125 * $toggle-size);
+}
+
+/* Hide default HTML checkbox */
+.switch input {
+  opacity: 0;
+}
+
+/* The toggle */
+.toggle {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: 0.4s;
+  transition: 0.4s;
+  border-radius: 34px;
+}
+
+.toggle:before {
```

Modern best practice is to use two colons for pseudo-elements, whereas `:` is reserved for "states" like `:checked`

```suggestion
.toggle::before {
```